hw/kdrive/src/kdrive.c

```diff
@@ -508,6 +513,10 @@ KdProcessArgument(int argc, char **argv, int i)
         kdSoftCursor = TRUE;
         return 1;
     }
+    if (!strcmp(argv[i], "-videoTest")) {
```

Comment:
probably should be separate commit. 
note that help text and already mentioning this.